### ISSUE TITLE
There's one thing that users should be made aware of when the update to this version:

### ISSUE THREAD

**Issue by Febbe** [xournalpp/xournalpp#3597]
There's one thing that users should be made aware of when the update to this version:

> In case you ordered your colors in the toolbar in a custom way this order is lost in the migration. However, if you reorder them afterwards this order will be persisted, so it's a one time effort.

When we do the release we might want to put this in the release notes.

_Originally posted by @idotobi in https://github.com/xournalpp/xournalpp/pull/2379#issuecomment-975204159_

**Comment by Technius** [comment]
Was this regarding 1.1.0 or 1.1.1?

**Comment by Febbe** [comment]
> Was this regarding 1.1.0 or 1.1.1?

1.1.1

**Comment by rolandlo** [comment]
> > Was this regarding 1.1.0 or 1.1.1?
> 
> 1.1.1

It looks like 1.2 to me. The customizable color palette are not part of 1.1.1.

**Comment by Febbe** [comment]
> > > Was this regarding 1.1.0 or 1.1.1?
> > 
> > 1.1.1
> 
> It looks like 1.2 to me. The customizable color palette are not part of 1.1.1.

Right somehow I got confused; there were only 2 choices and I only chose the newest. It's of cause from the current master which will be most likely 1.2

**Comment by TentativeConvert** [comment]
I recently lost my custom ([colourblind-friendly](https://davidmathlogic.com/colorblind/#%23D81B60-%231E88E5-%23FFC107-%23004D40)) colours following an update of xournalpp.  Could that be related to the warning in this thread?  I now have the following version:

    1.1.1+dev
    Built on Feb 14 2022, 10:39:46

I suppose this is the latest stable version as I'm only following the `apanda1` repository.  
With the previous version, I had custom colours specified in `toolbar.ini`, like so:

     toolbarLeft1=COLOR(0x332288),COLOR(0x117733),COLOR(0x44AA99), ...

With the new version, these colours seem to be ignored.  If I understand correctly, the new syntax should be:

    toolbarLeft1=COLOR(0),COLOR(1),COLOR(2),COLOR(3), ...

referring to certain colors defined in `palette.gpl`.  So I switched to the new syntax and edited the colors in `palette.pgl`, but these edits are likewise ignored.  Is this the expected behaviour in the current version?  Is there any workaround?

**Comment by idotobi** [comment]
@TentativeConvert : Thank you for reporting the issue :pray: 
Would it be OK for you to share your `palette.gpl` and the `toolbar.ini`?
This way I think I could find the problem rather quickly :nerd_face:

**Comment by TentativeConvert** [comment]
toolbar.ini:

    [Right hand Note Taking Copy]
    toolbarTop1=SAVE,NEW,OPEN,SEPARATOR,CUT,COPY,PASTE,SEPARATOR,UNDO,REDO,SEPARATOR,ERASER,PEN,HIGHLIGHTER,IMAGE,TEXT,MATH_TEX,SEPARATOR,DEFAULT_TOOL,SEPARATOR,INSERT_NEW_PAGE,DELETE_CURRENT_PAGE,SEPARATOR,GOTO_BACK,GOTO_NEXT,SEPARATOR,FULLSCREEN,SEPARATOR,AUDIO_RECORDING,AUDIO_SEEK_BACKWARDS,AUDIO_PAUSE_PLAYBACK,AUDIO_STOP_PLAYBACK,AUDIO_SEEK_FORWARDS
    toolbarLeft1=COLOR(7),COLOR(0),COLOR(1),COLOR(3),COLOR(2),COLOR(4),COLOR(5),COLOR(6),COLOR_SELECT,SEPARATOR,PRESENTATION_MODE,ZOOM_100,ZOOM_FIT,ZOOM_IN,ZOOM_OUT
    toolbarLeft2=FINE,MEDIUM,THICK,SEPARATOR,TOOL_FILL,SEPARATOR,DRAW_ELLIPSE,DRAW_RECTANGLE,DRAW_ARROW,RULER,SEPARATOR,ROTATION_SNAPPING,GRID_SNAPPING,SEPARATOR,VERTICAL_SPACE,SELECT_REGION,SELECT_RECTANGLE,SELECT_OBJECT,PLAY_OBJECT
    name=Right hand Note Taking Copy

palette.gpl:

    GIMP Palette
    Name: Xournal Default Palette
    #
    51 34 136 Dark Blue
    17 119 51 Green
    68 170 153 Teal
    136 204 238 Light Blue
    221 204 119 Ochre
    204 102 119 Magenta
    170 68 153 Violet
    136 34 85 Lavender
    255 255 255 White
    0 0 0 Black

(I tried uploading the files but these formats don't seem to be supported.)

**Comment by AnnaVd** [comment]
I have the same problem, Ubuntu 20.04 and xournalpp 1.1.1+dev.
Here is a snippet of ~/.xournalpp/toolbar.ini
```
toolbarLeft2=FINE,MEDIUM,THICK,VERY_THICK,SEPARATOR,SEPARATOR,RULER,DRAW_CIRCLE,DRAW_RECTANGLE,DRAW_ARROW,COLOR(0),COLOR(1),COLOR(2),SEPARATOR,SEPARATOR,PEN,HILIGHTER
```
and the entire ~/.xournalpp/palette.gpl
```
31 72 131 MyDarkBlue
210 93 208 MyPink
0 192 255 MyLightBlue
```
These custom colors seem to be ignored entirely, and some other random colors appear instead:
![image](https://user-images.githubusercontent.com/32481526/155152352-56885cc7-02a1-40c5-b674-2133282f37c1.png)

**Comment by TentativeConvert** [comment]
> Which Operating system are you using? On which path relative to home are the files located?

I am also on Ubuntu 20.04.  My files are located in ~/.config/xournalpp/

**Comment by AnnaVd** [comment]
> > Which Operating system are you using? On which path relative to home are the files located?
> 
> I am also on Ubuntu 20.04. My files are located in ~/.config/xournalpp/

Ah I solved my problem by modifying the files in ~/.config/xounalpp/ instead of ~/.xournalpp 
Don't know why it's not working for you though :(

**Comment by TentativeConvert** [comment]
My bad.  It wasn't working in my case because of an incorrect file extension.  Thank you for looking into this!

**Comment by bhennion** [comment]
We forgot about this when we released 1.2 I guess. Closing.
